pkg/reexec/reexec_deprecated.go

```diff
@@ -0,0 +1,41 @@
+// Package reexec facilitates the busybox style reexec of a binary.
+//
+// Deprecated: this package is deprecated and moved to a separate module. Use [github.com/moby/sys/reexec] instead.
+package reexec
+
+import (
+	"os/exec"
```

Comment:
Is this dependency required for the godoc link?

Reply:
It's needed for the aliases currently, as `exec.Cmd` is part of the signature;


```go
func Command(args ...string) *exec.Cmd {
```

The alternative is to use variables to alias, but I _slightly_ moved away from that approach, as pkg.go.dev docs will then show an empty "functions" block, and only "variables" in the package. Also because these are exported, it would mean technically anything could overwrite / replace the function, which definitely would be unwanted.

I once opened a proposal in Go's issue tracker for being able to alias a package as a whole, but they were not receptive to that;

- https://github.com/golang/go/issues/56611